packaging/linux/org.shotcut.Shotcut.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0"?>
+<mime-info xmlns='http://www.freedesktop.org/standards/shared-mime-info'>
+  <mime-type type="application/vnd.mlt-project">
```

Comment:
Shotcut uses "application/mlt+xml" to identify some things placed on the clipboard. And then, for whatever reason - probably lack of detailed scrutiny - Info.plist uses "text/mlt+xml". Let's switch everything over to "application/vnd.mlt+xml". I will take care of the Shotcut source code changes for that. "+xml" is preferred over "-xml" because it is a [suffix](https://en.wikipedia.org/wiki/Media_type#Suffix).